addon/styles/_frost-detail-subtab.scss

```diff
@@ -4,9 +4,6 @@ $frost-detail-tab-inactive: $frost-color-grey-5;
 
 .frost-detail-subtab {
   position: relative;
-  display: flex;
-  flex-direction: row;
-  justify-content: center;
```

Comment:
This was causing issues. It was resizing the icons. Everything is still centered.